client/app/components/dashboards/TextboxDialog.jsx

```diff
@@ -53,20 +62,21 @@ class AddTextboxDialog extends React.Component {
 
   render() {
     const { dialog } = this.props;
+    const isNew = !this.props.text;
 
     return (
       <Modal
         {...dialog.props}
-        title="Add Textbox"
+        title={isNew ? 'Add Textbox' : 'Edit Textbox'}
         onOk={() => this.saveWidget()}
         okButtonProps={{
           loading: this.state.saveInProgress,
           disabled: !this.state.text,
         }}
-        okText="Add to Dashboard"
+        okText={isNew ? 'Add to Dashboard' : 'Save'}
         width={500}
       >
-        <div className="add-textbox">
+        <div className="textbox-dialog">
```

Comment:
Need to update classname in `TextboxDialog.less` as well

Reply:
10x I missed that 😓 Fixed.